tests/integration/standard/test_single_interface.py

```diff
@@ -45,7 +46,7 @@ def tearDown(self):
             self.cluster.shutdown()
 
     # TODO: enable after https://github.com/scylladb/python-driver/issues/121 is fixed
-    @unittest.skip('Fails on scylla due to the broadcast_rpc_port is None')
+    @pytest.mark.skip('Fails on scylla due to the broadcast_rpc_port is None')
```

Comment:
Why it is None on scylla, is it expected ? If you don't want to look into it, can you please create an issue for it.

Reply:
there is an issue for it on the comment, you raise it yourself :)

Reply:
https://github.com/scylladb/python-driver/issues/121